Source/Particles/Collision/BinaryCollision/BinaryCollision.H

```diff
@@ -199,6 +217,12 @@ public:
                     amrex::HostDevice::Atomic::Add( &(*cost)[mfi.index()], wt);
                 }
             }
+
+            if (m_have_product_species) {
+                // Call redistribute to remove particles with negative ids
+                species1.Redistribute(lev, lev, 0, 0, true);
+                species2.Redistribute(lev, lev, 0, 0, true);
```

Comment:
The `local` flag should actually be `1`. See discussion at https://github.com/ECP-WarpX/WarpX/pull/4831#discussion_r1554498964
```suggestion
                species1.Redistribute(lev, lev, 0, 1, true);
                species2.Redistribute(lev, lev, 0, 1, true);
```